server/controllers/entities/lib/resolver/create_entity_from_seed.js

```diff
@@ -71,6 +77,8 @@ const createEntityFromSeed = params => {
   .then(addCreatedUriToSeed(seed))
 }
 
+// Difference between seedClaims and entityClaims:
+// seedClaims come from the request input, and might not be valid
```

Comment:
The difference is less the data quality than the data origin : seed being from an external source, while entity is the current inventaire entity

Reply:
`entityClaims` is not the claims of a currently existing entity, but the `claims` object created higher in the file, and being enriched. I refactored (29752066) to try to make that clearer